test/e2e/tests/edit-gas-fee.spec.js

```diff
@@ -0,0 +1,280 @@
+const { strict: assert } = require('assert');
+const {
+  connectDappWithExtensionPopup,
+  getWindowHandles,
+  largeDelayMs,
+  withFixtures,
+  regularDelayMs,
+} = require('../helpers');
+
+describe('Editing Confirm Transaction', function () {
+  it('allow selecting high, medium, low gas estimates on edit gas fee popover', async function () {
+    const ganacheOptions = {
+      hardfork: 'london',
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: '0x15af1d78b58c40000',
+        },
+      ],
+    };
+    await withFixtures(
+      {
+        fixtures: 'eip-1559-v2',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        if (process.env.EIP_1559_V2 === '1') {
```

Comment:
Seems that you could wrap the whole `it` block in this, yeah? Otherwise the server will start for nothing. Same goes for other EIP-1559 v2 tests.

Reply:
Fixed